Sources/Uno/OneTimePassword/Kind.swift

```diff
@@ -9,10 +9,34 @@ import Foundation
 public extension OneTimePassword {
   /// The supported types of One Time Passwords.
   enum Kind {
+    /// The key of the OTP type.
+    enum Key: String {
+      /// The key for the HOTP.
+      case hotp
+      
+      /// The key for the TOTP.
+      case totp
```

Comment:
```suggestion
      /// The key for the HOTP.
      case hOTP
      
      /// The key for the TOTP.
      case tOTP
```

Reply:
We decided to keep the lowercase name of the cases since they're acronyms.